elm-package.json

```diff
@@ -11,9 +11,8 @@
         "Bootstrap.Html.Internal"
     ],
     "dependencies": {
-        "circuithub/elm-html-shorthand": "11.0.0 <= v < 12.0.0",
```

Comment:
I'm happy to merge this, but of course it need elm-html-shorthand, pending https://github.com/circuithub/elm-html-shorthand/pull/13